ownCloud/Resources/Info.plist

```diff
@@ -60,7 +70,5 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
-	<key>ITSAppUsesNonExemptEncryption</key>
-	<false/>
```

Comment:
Why did you remove this?

Reply:
I think this was removed without notice in a merge/rebase, now it's in place again.